x-pack/plugins/security_solution/public/common/lib/apm/user_actions.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const SECURITY_ACTIONS_PREFIX = 'securitySolution';
```

Comment:
Nit: There's some discrepancy between this prefix and the one used for naming `route-change` transactions, where it's called `securitySolutionUI`. Could we use consistent naming?

<img width="820" alt="Screenshot 2022-06-08 at 19 58 52" src="https://user-images.githubusercontent.com/7359339/172685290-7dbbf1fb-4198-4420-8fad-616b7319088d.png">

<img width="1847" alt="Screenshot 2022-06-08 at 19 59 50" src="https://user-images.githubusercontent.com/7359339/172685588-b0eb7acb-966b-4252-a394-d01039ccf0cf.png">